src/server/lib/ab.ts

```diff
@@ -33,10 +41,14 @@ export const selectWithSeed = <V extends Variant>(
 };
 
 /**
+ * For use in AB tests.
  * If controlProportionSettings is set then we use this to define the range of mvt values for the control variant.
  * Otherwise we evenly distribute all variants across maxMvt.
  */
-export const selectVariant = <V extends Variant, T extends Test<V>>(test: T, mvtId: number): V => {
+export const selectVariantUsingMVT = <V extends Variant, T extends Test<V>>(
```

Comment:
banner + header selection is still done using this function (for AB testing only)